.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish Docker image
+
+on: push
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to GitHub Packages
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.CR_PAT }}
```

Comment:
Whoever's account will be used to create the PAT would need to have maintainer access to this repo I think.

Reply:
Thanks yah, we already have a bot that we use for most of our automatic things that we would use for this, thanks for pointing it out.